packages/lit-dev-content/samples/examples/motion-carousel/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<head>
+  <script type="module" src="./my-element.js"></script>
+  <style>
+    body {
+      margin: 0;
+      height: 100vh;
+      width: 100vw;
+      font-family: 'Open Sans', sans-serif;
```

Comment:
You'd need a google fonts tag to load Open Sans -- it won't be inherited from the iframe parent. I would probably just stick with `sans-serif` though to keep things simple, should look fine with whatever system default.

Reply:
Done.